deploy/cert-manager-webhook-dnspod/values.yaml

```diff
@@ -27,6 +27,8 @@ clusterIssuer:
   #    - 'example.com'
 
 image:
+  # imagePullSecrets:
```

Comment:
Move to root following https://github.com/kubernetes/ingress-nginx/blob/master/charts/ingress-nginx/values.yaml#L711